Tests/DatadogTests/Datadog/URLSessionAutoInstrumentation/DDURLSessionDelegateTests.swift

```diff
@@ -94,26 +94,28 @@ class DDURLSessionDelegateTests: XCTestCase {
 
         // When
         let taskWithURL = session.dataTask(with: URL.mockAny())
+        taskWithURL.taskDescription = "taskWithURL"
```

Comment:
As (in flaky scenario) we receive some other task than we expect, here I tag each task with a custom description, so we can see it in the error (expected vs actual `taskDescription`).